src/tes/config.go

```diff
@@ -26,44 +50,114 @@ type Openstack struct {
 
 // Schedulers describes configuration for all schedulers.
 type Schedulers struct {
-	Local     Local
-	Openstack Openstack
+	Local     Local     `json:",omitempty"`
+	Dumblocal Local     `json:",omitempty"`
+	Condor    Local     `json:",omitempty"`
+	Openstack Openstack `json:",omitempty"`
 }
 
 // Config describes configuration for TES.
 type Config struct {
-	pbr.ServerConfig
-	// TODO move this to protobuf?
-	Schedulers Schedulers
-	DBPath     string
-	HTTPPort   string
-	RPCPort    string
-	Scheduler  string
-	ContentDir string
-	WorkDir    string
+	ServerConfig ServerConfig
+	Scheduler    string
+	Schedulers   Schedulers
+	Worker       Worker
+	DBPath       string
+	HTTPPort     string
+	RPCPort      string
+	ContentDir   string
+	WorkDir      string
+	LogLevel     string
 }
 
 // DefaultConfig returns configuration with simple defaults.
 func DefaultConfig() Config {
 	workDir := "tes-work-dir"
 	return Config{
-		ServerConfig: pbr.ServerConfig{
+		ServerConfig: ServerConfig{
```

Comment:
Actually, since we've removed the protobuf messages, we can probably just ditch the `ServerConfig` struct and move all the fields to the root `Config` struct.